5eShaped/precompiled/translations/de.json

```diff
@@ -55,17 +55,17 @@
     "athletics": "Athletik",
     "deception": "Täuschung",
     "history": "Geschichte",
-    "insight": "Einsicht",
-    "intimidation": "Einschüchterung",
-    "investigation": "Investigation",
+    "insight": "Motiv erkennen",
+    "intimidation": "Einschüchtern",
```

Comment:
"Entdecken" equals "detection", where "Wahrnehmung" equals "perception"
"Einschüchtern" is the term used in 3.5. I did not think about whehter or nor we are looking at nouns versus verbs, but if you feel this is important, please keep "Einschüchterung".